README.md

```diff
@@ -43,6 +43,10 @@ The cookbook is a living document. :seedling:
 5. [Star Schema Denormalization (N-Way Join)](joins/05/05_star_schema.md)
 6. [Lateral Table Join](joins/06/06_lateral_join.md)
 
+### Set Operations
```

Comment:
I don't think this deserves a new category. How many more recipes will we write on set operations? As we discussed before, this is not supposed to fully emulate the docs. 😉 

What about expanding "Other Built-in Functions" to "Other Built-in Functions & Operators"?